### DISCUSSION TITLE
Modul Tabs / Accordion - how to edit / add content

### DISCUSSION THREAD

**Discussion by walhallaRV** [microweber/microweber#935] [General]
Hi,
I tried to put an accordion / tabs - modul on a site. OK, it works. But I only can add / edit / see the content of the first tab / accordion. I can add more, can see the tabs, but how can I add content to the new tabs / accordions.

Playing around for hours now and can't find how. Becoming realy crazy ;) On the website I can see the accordion / tabs with content in the first tab / accordion. But if i click on any other tab / accordion nothing happens. Doesn't collapse, not change anything.

I tried the same in the Microweber demo - site: there apears  a red pop-up (maybe errors?) but just 1 second and I can read so fast. Same problem: can see the accordion / tabs, can add / see content of the first one. But the rest is unvisible, cant input content.

Thanks so much for your help!

Cheers Walhalla

**Comment by peter-mw** [comment]
hi, this is a template bug that we fixed

the content is there , just the css is broken 

will release new version soon

**Reply by walhallaRV** [reply]
Hi Peter, do you have any idea when this template will be fixed? Thanks for your response!

**Comment by walhallaRV** [comment]
Hi, thank you for the response. We are looking for a customer (paying) to substitute their existing system. OK, we will wait some days. But one of the requirements is a working accordion / tabs because they love and need it.  And we can't present / propose this system change if it doesn't work and we can't explain that anything, any option wich is offered doesn't work. Hopefully and thanks for the answer.
